config/Dockerfiles/ostree_image_compose/ostree-image-compose.sh

```diff
@@ -99,13 +99,22 @@ imagefactory --debug --imgdir $imgdir --timeout 3000 base_image /home/output/log
 imgname="fedora-atomic-$version-$release"
 qemu-img convert -c -p -O qcow2 $imgdir/*body /home/output/images/$imgname.qcow2
 
-# delete images over 3 days old
-find /home/output/images/ -type f -mtime +3 -exec rm {} \;
+pushd /home/output/images || exit 1
+latest=""
+if [ -e "latest-atomic.qcow2" ]; then
+    latest=$(readlink latest-atomic.qcow2)
+fi
+
+# delete images over 3 days old but don't delete what our latest link points to
+find . -type f -mtime +3 ! -name "$latest" -exec rm {} \;
 
 # Also delete if we have more than 3 images (from manual runs)
-pushd /home/output/images
-ls -t | sed -e '1,3d' | xargs --no-run-if-empty -d '\n' rm
-ln -sf $imgname.qcow2 latest-atomic.qcow2
+# but don't delete what our latest link points to
+images=$(ls -t)
+if [ -n "$latest" ]; then
+    images=$(echo $images| grep -v "$latest")
+fi
+echo $images| sed -e '1,3d' | xargs --no-run-if-empty -d '\n' rm
```

Comment:
if these items are just files, should we add '-v' to rm to see what is it doing?